src/Sql/Sql/Database Backup/Services/AzureSqlDatabaseBackupAdapter.cs

```diff
@@ -390,24 +392,36 @@ private AzureSqlDatabaseLongTermRetentionBackupModel GetBackupModel(Management.S
                 Location = locationName,
                 ResourceId = backup.Id,
                 ServerCreateTime = backup.ServerCreateTime,
-                ServerName = backup.ServerName
+                ServerName = backup.ServerName,
+                ResourceGroupName = GetResourceGroupNameFromResourceId(backup.Id)
             };
         }
 
+        private string GetResourceGroupNameFromResourceId(string resourceId)
```

Comment:
There is a ResourceIdentifer helper class that you can use to help grab ResourceGroupName from resourceId.